splash/browser_tab.py

```diff
@@ -537,6 +537,13 @@ def http_get(self, url, callback, headers=None, follow_redirects=True):
             follow_redirects=follow_redirects
         )
 
+    def http_post(self, url, callback, headers=None, follow_redirects=True, body=None):
```

Comment:
seems like redirects after POST are already handled by PyQt if you use `go()` but indeed if we use `http_post` they are not handled, and request method after redirect is still POST not GET